projects/plugins/jetpack/modules/widgets/gallery.php

```diff
@@ -338,14 +358,19 @@ public function update( $new_instance, $old_instance ) {
 	 *
 	 * Helps keep things nice and secure by listing only allowed values.
 	 *
-	 * @param array $instance The Widget instance to sanitize values for
+	 * @param array $instance The Widget instance to sanitize values for.
 	 * @return array $instance The Widget instance with values sanitized
 	 */
 	public function sanitize( $instance ) {
 		$allowed_values = $this->allowed_values();
 		$defaults       = $this->defaults();
 
 		foreach ( $instance as $key => $value ) {
+			if ( ! is_string( $value ) ) {
```

Comment:
That's true, but I'm not seeing 'columns' as an option on the widget's settings form anyway 🤷 

`sanitize_text_field( $value );` is also called later, so I thought only considering strings here was a quick fix for the fatal I saw, without refactoring too much.

Reply:
Hmm. Yeah, as far as I can tell the columns setting has never been used. Setting UI for it was apparently present at one point on wpcom, but is currently commented out with a note that it doesn't do anything.

I wonder, should we just delete the references to it in the functions near here? Or since it seems the concern was arrays we could probably make this `! is_scalar( $value )` to let integers through too.

Reply:
> we could probably make this ! is_scalar( $value ) to let integers through too.

I like that approach 👍  3beb180373fb943772155aed93db9c64df2c243f